Externals.cfg

```diff
@@ -45,7 +45,7 @@ required = True
 protocol = git
 repo_url = https://github.com/NOAA-GSL/GSI.git
 # Specify either a branch name or a hash but not both.
-#branch = feature/rrfs
+#branch = feature/rrfs-cmake-refactor
```

Comment:
GSI refactoring is not merged yet?

Reply:
It was merged, but in this branch. Do not use rrfs branch for further development. Use this branch.